lib/liquid/combinators/tags/increment.ex

```diff
@@ -19,6 +19,8 @@ defmodule Liquid.Combinators.Tags.Increment do
   import NimbleParsec
   alias Liquid.Combinators.Tag
 
+  @type t :: [increment: [variable_name: String.t()]]
```

Comment:
Shouldn't an Increment type be just a tagged tuple instead of a list of tagged tuples?

Reply:
Every tag is a keyword list (which is a list of tagged tuples)